sphinx/writers/latex.py

```diff
@@ -446,6 +446,18 @@ def __init__(self, document, builder):
             self.elements['multilingual'] = '%s\n\\setmainlanguage{%s}' % \
                 (self.elements['polyglossia'], self.babel.get_language())
 
+        # detect Chinese language, disable babel/polyglossia
+        # give warnings when no common chinese package found in configuration
+        if builder.config.language:
+            if builder.config.language.split('_')[0] == 'zh':
+                self.elements['babel'] = ''
+                self.elements['polyglossia'] = ''
+                if 'ctex' not in self.elements['usepackages'].lower():
```

Comment:
maybe we can clear this warning and add some notes to user manual?
and, is it good to clear babel warnings for chinese?